content/docs/05.developer-guide/04.inputs.md

```diff
@@ -258,8 +258,6 @@ with open("/tmp/example.txt", 'rb') as fh:
   mp_encoder = MultipartEncoder(fields={
     "string": "a string",
     "optional": "an optional string",
-    "int": 1,
-    "float": 1.255,
```

Comment:
These 2 lines lead to error:
AttributeError: 'int' object has no attribute 'encode'
AttributeError: 'float' object has no attribute 'encode'

Hence, removed these.

Reply:
Please use
```
"int": str(1),
"float": str(1.255),
```
instead